src/mavenTerminal.ts

```diff
@@ -160,6 +160,10 @@ function toDefaultWslPath(p: string): string {
 }
 
 export async function toWslPath(filepath: string): Promise<string> {
+    if (filepath.startsWith("/")) { // unix-like absolute path

```

Comment:
How about `path.posix.isAbsolute()`

Reply:
I don't see much difference for `path.posix.isAbsolute()`, it's also checking the first char. See https://github.com/nodejs/node/blob/d0377a825bf7ceb838570f434fdd7d4b1773b8fa/lib/path.js#L358-L359

I'm ok with both, any other benefit bisides better readability?

Reply:
Yes it's for the readability, the comment is no-need then.